security_groups/dynamic_asgs.go

```diff
@@ -4,13 +4,15 @@ import (
 	"crypto/tls"
 	"fmt"
 	"io"
+	"io/ioutil"
```

Comment:
Can you please replace `ioutil` with the corresponding `io` or `os` functions. `ioutil` has been deprecated since go1.16.